SyncApp/Controllers/HomeController.cs

```diff
@@ -218,6 +218,10 @@ public async Task<ActionResult> ExportSalesAsync(bool fromWeb, DateTime dateToRe
 
             try
             {
+                GetCountriesFromShopify();
+                
+                var shopifyCountries = await _countriesLogic.GetCountriesAsync();
```

Comment:
Can you please remove this line as it is not needed here

Reply:
Done